README.md

```diff
@@ -294,7 +295,29 @@ For **∅ or more** and **1 or more** element(s) of the same type you can use th
 Union types are denoted with the pipe symbol, `|`:
 
 ```js
-(userInput: String|Number) => String|Number
+(userInput: String | Number) => String | Number
+```
+
+### Negated Types
+
+It is sometime easier and more informative to delimit a type by defining what it's not. The negation operator let's you exclude by substracting from `Any`.
+
+```js
+JSON::parse(String, reviver: Function)
+  => Boolean | Number | String | Object | Array | null,
+  throws SyntaxError
+
+// is less concise than
+
+JSON::parse(String, reviver: Function)
+  => !Function & !Void & !Symbol,
+  throws SyntaxError
+
+// which is equivalent to
+
+JSON::parse(String, reviver: Function)
+  => !(Function & Void & Symbol),
+  throws SyntaxError
 ```
```

Comment:
> This example

Are you talking about [this one](https://github.com/ericelliott/rtype/pull/140/files#diff-04c6e90faac2675aa89e2176d2eec7d8R318)?